lib/bootstrap.php

```diff
@@ -27,6 +27,10 @@
     }
 }
 
+if (isset($dw_config['redis']) && !empty($dw_config['redis']['host'])) {
+    init_redis($dw_config['redis']['host'], $dw_config['redis']['port'], $dw_config['redis']['password']);
```

Comment:
since the Redis class is global as well I think we can ditch the one-liner init function

```suggestion
    Redis::init($host, $port, $password);
```